wasm-engine/src/lib.rs

```diff
@@ -0,0 +1,56 @@
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+use serde_wasm_bindgen::from_value;
+
+use unleash_yggdrasil::{
+    state::EnrichedContext, strategy_parsing::compile_rule, Context as YggdrasilContext,
+};
+use wasm_bindgen::prelude::*;
+
+#[derive(Deserialize, Serialize)]
+pub struct Context {
+    #[serde(rename = "userId")]
+    pub user_id: Option<String>,
+    #[serde(rename = "sessionId")]
+    pub session_id: Option<String>,
+    #[serde(rename = "environment")]
+    pub environment: Option<String>,
+    #[serde(rename = "appName")]
+    pub app_name: Option<String>,
+    #[serde(rename = "currentTime")]
+    pub current_time: Option<String>,
+    #[serde(rename = "remoteAddress")]
+    pub remote_address: Option<String>,
+    #[serde(rename = "groupId")]
+    pub group_id: Option<String>,
+    #[serde(rename = "properties")]
+    pub properties: Option<HashMap<String, String>>,
```

Comment:
#[serde(rename_all = "camelCase")] and skip the explicit rename for all of them?

Reply:
Ooo thanks! No idea why I went the long way here, applied rename_all